app/views/spree/recipes/taxons/index.html.erb

```diff
@@ -1,63 +1,16 @@
-                          <% if sub_category.icon.present? %>
-                              <%= image_tag main_app.url_for(sub_category.icon.try(:attachment)), alt:"#{sub_category.name} Image" %> 
-                          <% else %>
-                              <%= image_tag "noimage/default-category-image-small.jpg", alt:"#{sub_category.name} Image" %>
-                          <% end %>
-                        <% else %>
-                          <% if sub_category.icon.present? %>
-                              <%= image_tag main_app.url_for(sub_category.icon.my_cf_image_url(:large)), alt:"#{sub_category.name} Image" %> 
-                          <% else %>
-                              <%= image_tag "noimage/default-category-image.jpg", alt:"#{sub_category.name} Image" %>
-                          <% end %>
-                        <% end %>
-                      <% end %>
-
-                      <h4>
-                        <%= link_to sub_category.name.upcase, nested_recipe_taxons_path(sub_recipe_id), data: { turbolinks: false }, 'aria-label' => "List all #{sub_category.name}" %>
-                      </h4>
-                      <p>
-                        <%= raw(sub_category.short_description) %>
-                      </p>
-                      <% if sub_category.children %>
-                        <ul>
-                          <% sub_category.children.each do |subcategory| %>
-                            <% sub_recipe_id = subcategory.permalink.sub("recipes/", "") %>
-                            <li><%= link_to subcategory.name, nested_recipe_taxons_path(sub_recipe_id), data: { turbolinks: false }, 'aria-label': "Subcategory #{subcategory.name}" %></li>
-                          <% end %>
-                        </ul>
-                      <% end %>
-                      <%= link_to nested_recipe_taxons_path(sub_recipe_id), class: 'btn btn-primary', data: { turbolinks: false }, 'aria-label' => "See All #{sub_category.name} Products" do %>
-                        <span>See All <%= sub_category.name %></span>
-                      <% end %>
-                    </div>
-                  </div>    
-                <% end %>  
-              <% end %>
-            </div>
-          <% end %>  
-        </div>
+  <div class="recipes-hero-banners">
+    <div class="container">
+      <div class="row">
+        <%= render "slider", recipe_slide_locations: @recipe_slide_locations if @recipe_slide_locations.present? %>
       </div>
     </div>
   </div>
-</div>
+  <div class="category-search-wrapper">
+    <%= render "top_eight_categories", recipe_taxons: @top_eight_categories if @top_eight_categories.present? %>
+  </div>
+  
+  <div class="popular-categories-wrapper">
+    <%= render "top_four_popular_recipes", recipe_taxons: @top_four_categories if @top_four_categories.present? %>
+  </div>
+ <%= render "feature_recipe_video", recipe: @first_recipe %>
```

Comment:
why use first @first_recipe here?

Reply:
"Get Recipe" can point to any recipe for now. I will change this for staging/production. It will be static in the code.